Services/Infrastructure/OJS.Services.Infrastructure/HttpClients/Implementations/DistributorHttpClientService.cs

```diff
@@ -20,6 +21,13 @@ public DistributorHttpClientService(HttpClient client, ILogger<HttpClientService
                     string.Format(ValueCannotBeNullOrWhiteSpaceTemplate, SulsPlatformBaseUrlKey));
             }
 
+            var hostOriginHeaderValue = EnvironmentUtils.GetByKey(ApplicationUrl);
+            if (string.IsNullOrWhiteSpace(hostOriginHeaderValue))
```

Comment:
Check if this can be a validation on app start. We should not let the user make/retest submissions if we don't have key configurations available

Reply:
Done.

Reply:
Done.